packages/animation/src/extendAnimator/extendAnimator.test.tsx

```diff
@@ -16,11 +16,11 @@ test('Should extend actual animated component', () => {
     animator: AnimatorProvidedSettings
   }
   const ExampleComponent: FC<ExampleExtendAnimatorProps> = props => {
-    flow = props.animator.flow;
+    flow = props.animator?.flow;
     return null;
   };
-  const ExampleNode = withAnimator()(ExampleComponent);
-  const ExampleExtendedNode = extendAnimator({})(ExampleNode);
+  const ExampleNode = withAnimator<typeof ExampleComponent>()(ExampleComponent);
+  const ExampleExtendedNode = extendAnimator<typeof ExampleNode>({})(ExampleNode);
```

Comment:
For function components, yes. One could otherwise do something like:

```ts
  type ExampleComponentType = FC<ExampleExtendAnimatorProps>;
  const ExampleComponent: ExampleComponentType = props => {
    flow = props.animator?.flow;
    return null;
  };

  const ExampleNode = withAnimator<ExampleComponentType>()(ExampleComponent);
  const ExampleExtendedNode = extendAnimator<typeof ExampleNode>({})(ExampleNode);
```